fb_clone/Gemfile

```diff
@@ -40,48 +42,50 @@ gem 'jbuilder', '~> 2.5'
 
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
-gem 'foundation-rails'
+gem 'bootstrap-will_paginate'
+gem 'carrierwave'
 gem 'faker'
+gem 'foundation-rails'
+gem 'mini_magick'
 gem 'rubocop-faker'
-gem 'carrierwave'           
-gem 'mini_magick'           
-gem 'will_paginate'        
-gem 'bootstrap-will_paginate'
+gem 'will_paginate'
+gem 'font-awesome-rails'
 
 group :production do
-  gem 'pg', '>= 0.18', '< 2.0'
   gem 'fog'
+  gem 'pg', '>= 0.18', '< 2.0'
```

Comment:
You have declared `pg` gem twice.
You should probably keep only one of them.

Reply:
Fixed